ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -1,55 +1,57 @@
 import React, { useEffect, useState } from 'react';
-import { Switch, Route, useHistory } from 'react-router-dom';
-import { useDispatch, useSelector } from 'react-redux';
+import { Switch, Route, useHistory, useLocation } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 import Unlock from '../unlock-page';
 import {
-  DEFAULT_ROUTE,
-  ONBOARDING_ROUTE,
-  ONBOARDING_GET_STARTED_ROUTE,
-  ONBOARDING_HELP_US_IMPROVE_ROUTE,
+  // DEFAULT_ROUTE,
+  // ONBOARDING_ROUTE,
+  // ONBOARDING_GET_STARTED_ROUTE,
+  // ONBOARDING_HELP_US_IMPROVE_ROUTE,
   ONBOARDING_CREATE_PASSWORD_ROUTE,
-  ONBOARDING_IMPORT_WITH_SRP_ROUTE,
-  ONBOARDING_IMPORT_MOBILE_ROUTE,
-  ONBOARDING_SECURE_YOUR_WALLET_ROUTE,
+  // ONBOARDING_IMPORT_WITH_SRP_ROUTE,
+  // ONBOARDING_IMPORT_MOBILE_ROUTE,
+  // ONBOARDING_SECURE_YOUR_WALLET_ROUTE,
   ONBOARDING_REVIEW_SRP_ROUTE,
   ONBOARDING_CONFIRM_SRP_ROUTE,
-  ONBOARDING_PRIVACY_SETTINGS_ROUTE,
+  // ONBOARDING_PRIVACY_SETTINGS_ROUTE,
   ONBOARDING_COMPLETION_ROUTE,
   ONBOARDING_UNLOCK_ROUTE,
 } from '../../helpers/constants/routes';
-import OnboardingFlowSwitch from './onboarding-flow-switch/onboarding-flow-switch';
-import CreatePassword from './create-password/create-password';
-import SecureYourWallet from './secure-your-wallet/secure-your-wallet';
-import RecoveryPhrase from './recovery-phrase/confirm-recovery-phrase';
-import ConfirmRecoveryPhrase from './recovery-phrase/confirm-recovery-phrase';
-import StepProgressBar, {
-  stages,
-} from '../../components/app/step-progress-bar';
-import {
-  getCompletedOnboarding,
-  getIsInitialized,
-  getIsUnlocked,
-  getSeedPhraseBackedUp,
-} from '../../selectors';
+// import {
```

Comment:
I'm kindof leaving some stuff in there because we will use it as we continue to add the components here. We can remove them and add back but because this is just a WIP branch that we're stacking on top of I was thinking we could leave for now? I don't mind either way